tubular/nominal.py

```diff
@@ -26,6 +28,7 @@ def check_mappable_rows(self, X: pd.DataFrame) -> None:
             the mapping dict in mappings[c].
 
         """
+        self.columns_check(X)
```

Comment:
Good point, it was leading to not very helpful error messages in [this test](https://github.com/lvgig/tubular/blob/7b8f03e1c0df45e3aecdb2fa698ba3c24d42adb3/tests/base_tests.py#L249) when columns_check() was not run first which is why I added it. So rather than having columns_check() twice I've move the BaseTransformer.transform method first in [this commit](https://github.com/lvgig/tubular/blob/7b8f03e1c0df45e3aecdb2fa698ba3c24d42adb3/tests/base_tests.py#L249) instead